services/user_api/queries.js

```diff
@@ -0,0 +1,82 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'admin',
+  host: 'localhost',
+  database: 'CS3219',
+  password: 'password',
+  port: 5432,
+})
+
+
+const getUsers = (request, response) => {
+    pool.query('SELECT * FROM users ORDER BY userId ASC', (error, results) => {
+      if (error) {
+        throw error
+      }
+      response.status(200).json(results.rows)
+    })
+  }
+
+const getUserById = (request, response) => {
+const id = parseInt(request.params.id)
+
+pool.query('SELECT * FROM users WHERE userId = $1', [id], (error, results) => {
+    if (error) {
+    throw error
+    }
+    response.status(200).json(results.rows)
+})
+}
+
+
+const createUser = (request, response) => {
+const { username, password, email, createdDateTime} = request.body
```

Comment:
For the createdDateTime, the database side will automatically add with every new user, so I will pass to backend "null",

Also might want to consider hashing the password before adding to database
https://www.npmjs.com/package/bcryptjs
found a library you can consider using, see if you like

Reply:
Regarding the createdDateTime, I'll let the backend side handling parsing it. Also, I added your suggestion of hashing the password before adding to the daabase.